po/cs.po

```diff
@@ -26,147 +26,143 @@ msgstr "Odeslat heslo zpět do systému"
 
 #: ../src/core/org.freedesktop.systemd1.policy.in.in.h:2
 msgid "Authentication is required to send the entered passphrase back to the system."
-msgstr "Autentizace je vyžadována pro odeslání zadaného hesla do systému."
+msgstr "Pro odeslání zadaného hesla do systému je vyžadováno ověření."
 
 #: ../src/core/org.freedesktop.systemd1.policy.in.in.h:3
 msgid "Manage system services or other units"
-msgstr "Správa systémových služeb nebo dalších jednotek"
+msgstr "Spravovat systémové služby nebo další jednotky"
```

Comment:
The word "disk" is used for hard disks or optical drive disks, definitely not for unit files. Regarding your second question, I changed "Autentizace je vyžadována pro ..." to "...  je vyžadováno ověření." because the second one is used in most Czech translations.